npm-packages/convex/src/bundler/index.ts

```diff
@@ -297,10 +297,26 @@ export async function bundleSchema(
   dir: string,
   extraConditions: string[],
 ) {
-  let target = path.resolve(dir, "schema.ts");
-  if (!ctx.fs.exists(target)) {
-    target = path.resolve(dir, "schema.js");
+  const candidates = [
```

Comment:
would love to hear if people are using these extensions and why, but if this is helpful we can make this change. I'd guess the various TypeScript extensions would be more important? .mts, .cts, etc than .js, since most users use TypeScript? But I haven't heard requests for either

Reply:
This is fine if we remove .cjs, we never want to bundle a commonjs schema.